guby/settings.py

```diff
@@ -137,4 +137,47 @@
 
 BOOTSTRAP4 = {
     'include_jquery': True,
-}
\ No newline at end of file
+}
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'handlers': {
+        'console': {
+            'class': 'logging.StreamHandler',
+        },
+        'file': {
+            'level': 'DEBUG',
+            'class': 'logging.FileHandler',
```

Comment:
Instead of having a single file, do you think it makes more sense for us to have a handler that rotates the log file everyday? And keeps a 7 day log instead? 

It would also make it easier to track down problems and inspect them based on which day it occurred.